{"name": "test_register[test-15320254463-男-15320254463-sada@qq.com-False-10004-账号已存在！]", "status": "failed", "statusDetails": {"message": "AssertionError: 接口返回码是 【 200 】, 返回信息：操作成功 \nassert True == False\n +  where True = <service.result_base.ResultBase object at 0x000001DCC5331EE0>.success", "trace": "self = <testcases.api_test.index_uesr_test.test_01_register.TestRegitser object at 0x000001DCC52261F0>, userName = 'test', password = '15320254463', gender = '男', phone = '15320254463', email = 'sada@qq.com', expect_result = False\nexpect_code = 10004, expect_msg = '账号已存在！'\n\n    @allure.story(\"用例--用户注册\")\n    @allure.description(\"该用例是针对用户注册接口的测试\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"userName, password, gender, phone, email, expect_result, expect_code, expect_msg\",\n                             api_data[\"test_register\"])\n    def test_register(self, userName, password, gender, phone, email, expect_result, expect_code, expect_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = register(userName, password, gender, phone, email)\n        logger.info(\"*************** 成功执行后清理该数据 ***************\")\n        db.execute_db(f\"delete from user where phone={phone}\")\n>       assert_results(result,expect_result, expect_code, expect_msg)\n\ntest_01_register.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresult = <service.result_base.ResultBase object at 0x000001DCC5331EE0>, expect_result = False, expect_code = 10004, expect_msg = '账号已存在！'\n\n    def assert_results(result, expect_result, expect_code, expect_msg):\n        \"\"\"\n        断言结果并记录日志。\n        参数:\n            result: 实际结果对象，应该包含success, error, data等属性。\n            expect_result: 期望的成功状态。\n            expect_code: 期望的返回code。\n            expect_msg: 期望的返回message。\n        \"\"\"\n    \n        # 断言实际结果与期望结果相同\n>       assert result.success == expect_result, result.error\nE       AssertionError: 接口返回码是 【 200 】, 返回信息：操作成功 \nE       assert True == False\nE        +  where True = <service.result_base.ResultBase object at 0x000001DCC5331EE0>.success\n\n..\\..\\conftest.py:24: AssertionError"}, "description": "该用例是针对用户注册接口的测试", "attachments": [{"name": "log", "source": "4edece3a-f7ad-4d6d-a124-eec8de03b3cd-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "userName", "value": "'test'"}, {"name": "password", "value": "'15320254463'"}, {"name": "gender", "value": "'男'"}, {"name": "phone", "value": "'15320254463'"}, {"name": "email", "value": "'sada@qq.com'"}, {"name": "expect_result", "value": "False"}, {"name": "expect_code", "value": "10004"}, {"name": "expect_msg", "value": "'账号已存在！'"}], "start": 1729670192879, "stop": 1729670195402, "uuid": "edc92922-4b8a-4a0e-86e3-2c23f9ef2620", "historyId": "a86cd29c84935400f529258454bee51c", "testCaseId": "ce1de6727adff9873f648da1a6b8a7ea", "fullName": "testcases.api_test.index_uesr_test.test_01_register.TestRegitser#test_register", "labels": [{"name": "epic", "value": "针对单个接口的测试"}, {"name": "story", "value": "用例--用户注册"}, {"name": "feature", "value": "用户公共基础模块"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test.index_uesr_test"}, {"name": "suite", "value": "test_01_register"}, {"name": "subSuite", "value": "TestRegitser"}, {"name": "host", "value": "CQCA202413058N"}, {"name": "thread", "value": "21540-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.index_uesr_test.test_01_register"}]}